.teamcity/tests/buildTypes/E2ETests.kt

```diff
@@ -15,35 +14,30 @@ object E2ETests : BuildType({
     root(vcsRoots.KotlinLangOrg)
   }
 
-  steps {
-    script {
-      scriptContent = "./scripts/test/up.sh"
-    }
+    dependencies {
+        artifacts(BuildReferenceDocs) {
+            cleanDestination = true
+            artifactRules = "+:docs.zip!** => dist/docs/"
+        }
 
-    script {
-      scriptContent = "./scripts/test/run.sh"
     }
 
-    script {
-      scriptContent = "./scripts/test/stop.sh"
-    }
+  steps {
+      script {
+          scriptContent = "docker compose -f docker-compose-e2e-statics.yml up --exit-code-from playwright"
```

Comment:
I'd recommend putting script content in a file and calling it from there. 
That gives you the ability to highlight the script and set the flags.

Reply:
✅